module/platform/Gazebo/src/Gazebo.cpp

```diff
@@ -49,10 +49,10 @@ namespace module::platform {
             // Only listen to our target simulation
             if (network_source.name == config.simulator_name) {
                 if (real_time != 0 || sim_time != 0) {
-                    double real_delta          = simulation.real_time - real_time;
-                    double sim_delta           = simulation.simulation_time - sim_time;
-                    utility::clock::custom_rtf = utility::clock::custom_rtf * config.clock_smoothing
-                                                 + (1.0 - config.clock_smoothing) * (real_delta / sim_delta);
+                    double real_delta = simulation.real_time - real_time;
+                    double sim_delta  = simulation.simulation_time - sim_time;
+                    config.rtf = config.rtf * config.clock_smoothing + (1.0 - config.clock_smoothing) * (real_delta / sim_delta);
+                    //utility::clock::update_rtf(rtf);
```

Comment:
@PiiDish why is this commented,  we should be updating the custom clock rtf here (change `rtf` to `config.rtf`).